db/migrate/20190214125845_add_comments_to_miq_tasks_table.rb

```diff
@@ -0,0 +1,21 @@
+class AddCommentsToMiqTasksTable < ActiveRecord::Migration[5.0]
+  def up
+    change_table_comment :miq_tasks, "ManageIQ background tasks"
+
+    change_column_comment :miq_tasks, :context_data, "Serialized metadata that may contain more detailed information about the task."
+    change_column_comment :miq_tasks, :created_on, "The timestamp the task was added to the app inventory."
```

Comment:
I do not think we are updating this column explicitly, do we need to comment on default behavior ?

Reply:
It doesn't hurt. On a side note, I've noticed minor differences between tables, e.g. `created_at` vs `created_on`. Does rails treat this as synonyms?

Reply:
Yes, Rails recently switched from `created_on` to `created_at`.  Maybe we should add a prefix like `Builtin` or more specifically `Rails`?  Then it could say `"Builtin: the timestamp that the record was created"`

Reply:
Well, we haven't done that in the past, and it wouldn't make sense to other applications that might be referencing this table (if that ever happens). I'd like to just leave it if it's all the same.

Reply:
We don't have much of a precedent yet (one other table documented).
I agree with @yrudman that it is different from other columns in the table in that our app doesn't "own" (and shouldn't change) this column, so our documentation should probably reflect that in some way.

Reply:
Well, there's also the vms table from a short while back. Ok, I'll start tagging those columns with "builtin".

Reply:
Ok, tagged.